### PR TITLE
Instrument seminar series page

### PR DESCRIPTION
**Pull request by y4izus**
Part of #1374

### CODE REVIEW

**Review by eddybrando — APPROVED** [review]
Awesome! This will be very helpful for our analytics. 🙂

Thanks @y4izus!

File: pages/events/seminar-series.vue [hunk 1]
```diff
@@ -37,31 +37,43 @@ export default class SeminarSeriesPage extends QiskitPage {
...
+        }
       }
     },
     {
       title: 'Code of conduct',
       description: 'Qiskit is dedicated to providing an enjoyable and safe experience for all participants. We have a code of conduct that all events adhere to.',
       cta: {
         url: 'https://github.com/Qiskit/qiskit/blob/master/CODE_OF_CONDUCT.md',
-        label: 'See code of  conduct'
+        label: 'See code of  conduct',
```
Comment (techtolentino) [inline pages/events/seminar-series.vue:73]: small extra space here

**Review by techtolentino — APPROVED** [review]
The segment update looks good, @y4izus. Small extra space is my only feedback.

File: pages/events/seminar-series.vue (hunk 1 above)
Comment (y4izus) [reply pages/events/seminar-series.vue:73]: Good catch Randy! thanks!

**Pull request merged**